[FIX]sale_stock_ux: qty_to_invoice

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/sale#1340

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/1415.png)](https://mergebot.adhoc.ar/ingadhoc/sale/pull/1415)

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc cherrypicking of pull request ingadhoc/sale#1340 failed.

stdout:
```
Auto-merging sale_stock_ux/models/sale_order_line.py
Auto-merging sale_stock_ux/models/stock_move.py
CONFLICT (content): Merge conflict in sale_stock_ux/models/stock_move.py

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

:warning: after resolving this conflict, you will need to merge it via @roboadhoc.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).

jcadhoc committed:
[FIX] sale_stock_ux: qty_to_invoice

X-original-commit: 69e2e318e1e980ba1f092d20e3adaeae57365b00

# Conflicts:
#	sale_stock_ux/models/sale_order_line.py
#	sale_stock_ux/models/stock_move.py

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).

fw-bot-adhoc commented:
@jcadhoc @maq-adhoc this forward port of ingadhoc/sale#1340 is awaiting action (not merged or closed).
